src/routes/groups.js

```diff
@@ -15,13 +15,15 @@ async function fetchGroup (platform, urlname) {
     })
 }
 
-async function blackListGroup (group) {
+async function blackListGroup (group, newState) {
   await group.update({
-    blacklisted: true
+    blacklisted: newState
   })
 
+  const shouldBeActive = !newState
+
   return db.sequelize.query(
-    'UPDATE "Events" SET active = false WHERE platform = ? AND group_id = ?',
+    `UPDATE "Events" SET active = ${shouldBeActive} WHERE platform = ? AND group_id = ?`,
```

Comment:
I tried using a `?` here, but it didn't let me. (I can't remember the exact error message.)

Reply:
You probably need to add it in as the first item in the object. 2nd arg for `query`.